packages/mui-material/src/Input/inputClasses.ts

```diff
@@ -5,14 +5,10 @@ import { inputBaseClasses } from '../InputBase';
 export interface InputClasses {
   /** Styles applied to the root element. */
   root: string;
-  /** Styles applied to the root element if the component is a descendant of `FormControl`. */
-  formControl: string;
   /** Styles applied to the root element if the component is focused. */
   focused: string;
   /** Styles applied to the root element if `disabled={true}`. */
   disabled: string;
-  /** Styles applied to the root element if color secondary. */
-  colorSecondary: string;
```

Comment:
Instead of adding the class to the DOM, I chose to remove it here. The fact that no users have reported this issue, even though it has likely existed for a long time (possibly since v5), suggests that it is not widely used. Therefore, I decided to remove the class instead of adding it to the DOM.